docs/UpdatedData.md

```diff
@@ -0,0 +1,209 @@
+{
+  "time": "2020-05-12T08:50:08.000Z",
+  "timezoneOffset": 120,
+  "deviceTime": "2020-05-12T08:50:08",
+  "deviceId": "IdOfTheDevice",
+  "type": "bolus",
+  "subType": "normal",
+  "normal": 3.5,
+  "expectedNormal": 4.0, 
+  "prescriptor": "system-altered"
+}
+```
+
+## physical activity
+
+We need additional fields to get the time at which the physical activity is created, and the last time it was updated by the patient:
+- `entryTime` is a UTC string timestamp that defines at what time the patient has entered the physical activity. This field is optional. It takes the same format as `time` field.
+- `lastUpdatedTime` is a UTC string timestamp that gives the last time the patient has updated the physical activity. This field is optional. It takes the same format as `time` field.
+  - `lastUpdatedTime` >= `entryTime`
+
+```json
+{
+    "type": "physicalActivity",
+    "reportedIntensity": "medium",
+    "duration": { 
+    	"value": 60,
+    	"units": "minutes"
+    },
+    "clockDriftOffset": 0,
+    "conversionOffset": 0,
+    "deviceId": "DexG5MobRec_DX72101079",
+    "deviceTime": "2016-07-12T23:52:47",
+    "entryTime": "2020-05-12T08:00:08.000Z",
+    "lastUpdatedTime": "2020-05-12T08:30:08.000Z",
+    "time": "2020-05-12T08:50:08.000Z",
+    "timezoneOffset": 60
+}
+```
+
+## Alarm events
+Leveraging the `deviceEvent` type with the already defined `alarm` subType. We add couple of fields to get more details on alarms and acknowledgement. 
+
+- `alarmLevel`: `alarm | alert` 
+- `alarmCode`: code of the alarm. This field is optional. 
+- `alarmLabel`: label or description of the alarm. This field is optional. 
+- `acknowledgeTime`: is a UTC string timestamp that indicates at what time the patient has acknowledged the alam. 
+
+```json
+{
+  "type": "deviceEvent",
+  "subType": "alarm",
+  "alarmType": "handset",
+  "alarmLevel": "alarm", 
+  "alarmCode": "123456",
+  "alarmLabel": "Label of the alarm",
+  "acknowledgeTime": "2020-05-12T08:55:00.000Z",
+  "deviceId": "Id12345",
+  "deviceTime": "2018-02-01T00:00:00",
+  "guid": "e3c82e3d-23ba-4048-9056-7b1b3c5aa4cc",
+  "id": "d5ed640dd8f74e6cb1a6bff796de3ba2",
+  "time": "2020-05-12T08:50:08.000Z",
+  "timezoneOffset": 60
+}
+```
+
+
+## Zen mode
+Leveraging the `deviceEvent` type and creating a new `zen` subType.
+
+- `subType`: `zen`
+- `endTime`: is a UTC string timestamp that indicates at what time the event is terminated. 
+
+```json
+{
+  "type": "deviceEvent",
+  "subType": "zen",
+  "endTime": "2020-05-12T09:50:08.000Z",
+  "deviceId": "Id12345",
+  "deviceTime": "2018-02-01T00:00:00",
+  "guid": "e3c82e3d-23ba-4048-9056-7b1b3c5aa4cc",
+  "id": "d5ed640dd8f74e6cb1a6bff796de3ba2",
+  "time": "2020-05-12T08:50:08.000Z",
+  "timezoneOffset": 60
+}
+```
+
+## Confidential mode
+Leveraging the `deviceEvent` type and creating a new `confidential` subType.
+
+- `subType`: `confidential`
+- `endTime`: is a UTC string timestamp that indicates at what time the event is terminated. 
```

Comment:
Same remark than the one the "lastUpdatedTime" field in physical activity.
We could store two object the first one with eventType="start" and a second one with the eventType="end", for example.

Reply:
same as above